services/brig/src/Brig/App.hs

```diff
@@ -575,6 +580,15 @@ newtype HttpClientIO a = HttpClientIO
       MonadIndexIO
     )
 
+type BrigEffects =
```

Comment:
I'd suggest calling this sometihng like `BrigCanonicalEffects` to highlight that other choices are possible

Reply:
Done in bba1c0998.